githooks/post-merge

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# === RUST-BITCOIN GITHOOK ===
+
+# Scan the githooks after merge. If the user is using any of the rust-bitcoin githooks and they have changed, let them know
+# to rerun the githooks setup.
+
+# We only care about this on master
+[[ "$(git branch --show-current)" = "master" ]] || exit
+
+# Get the local githooks directory, regardless of configuration.
+GIT_DIR=$(git rev-parse --git-common-dir)
+HOOKS_DIR=$(git config --get core.hooksPath || echo "$GIT_DIR/hooks")
+
+# Check if the user has opted to ignore the warning
+[[ -f "$HOOKS_DIR"/.ignore-scan ]] && exit
```

Comment:
Isn't this better achieved by removing (not installing) the post-merge hook?

Reply:
I prefer this for... reasons? 😆

I can't really form a coherent counter-argument. If I had to make a case, I would say that if something unrelated to scanning githooks gets added to the post-merge hook, something like this might need to be re-added so they can use that without deleting the entire hook to get rid of this warning.

Reply:
Yes but does it make sense to have it if it's not needed yet?

Also maybe a more interesting approach would be to have a config to warn only once and then this script sets "warned already" marker and the update script removes it.

Reply:
I updated to let people know to remove the post-merge githook if they don't want to see the warning anymore.